google-cloud-examples/src/main/java/com/google/cloud/examples/bigquery/snippets/dataset/CreateDataset.java

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.cloud.examples.bigquery.snippets.dataset;
+
+// [START bigquery_create_dataset]
+
+import com.google.cloud.bigquery.BigQuery;
+import com.google.cloud.bigquery.BigQueryException;
+import com.google.cloud.bigquery.BigQueryOptions;
+import com.google.cloud.bigquery.Dataset;
+import com.google.cloud.bigquery.DatasetId;
+import com.google.cloud.bigquery.DatasetInfo;
+
+public class CreateDataset {
+
+  public static Dataset createDataset(String projectId, String datasetName) {
```

Comment:
Any function parameters should have example declarations in the sample. (See https://github.com/GoogleCloudPlatform/java-docs-samples/blob/master/SAMPLE_FORMAT.md#function-parameters)

Currently we are doing this with comments:
```java
public static Dataset createDataset(String projectId, String datasetName) {
    // String projectId = "your-project-id"
    // String datasetName = "your-dataset-name"
```

But it's been suggested that we change to using a runner so that they pass compile and style checks:
```java
public static void runCreateDataset() {
    String projectId = "your-project-id";
    String datasetName = "your-dataset-name";
}
public static Dataset createDataset(String projectId, String datasetName) {
```